app/src/main/kotlin/com/wire/android/ui/userprofile/image/AvatarPickerViewModel.kt

```diff
@@ -59,7 +59,7 @@ class AvatarPickerViewModel @Inject constructor(
             withContext(Dispatchers.IO) {
                 val data = imgUri.toByteArray(context)
                 val mimeType = imgUri.getMimeType(context) ?: DEFAULT_IMAGE_MIME_TYPE
-                val result = uploadUserAvatar(mimeType = mimeType, imageData = data)
+                val result = uploadUserAvatar( imageData = data)
```

Comment:
```suggestion
                val result = uploadUserAvatar(imageData = data)
```

aaaand 🙌🏻  no more erros with unmatching parameter